README.md

```diff
@@ -181,6 +182,19 @@ Optional steps:
 [bash completions]: extra/completions/liquidctl.bash
 
 
+## Installing on FreeBSD
+
+liquidctl is maintained in the Ports Collection (thanks to ehaupt@FreeBSD.org), and it is available as a pre-built binary package.
+
+- port: `sysutils/py-liquidctl`
+- binary: `pkg install py37-liquidctl`
+- dependencies: `devel/py-docopt`, `comms/py-hidapi`, `devel/py-pyusb`
+
+liquidctl requires root privileges (doas or sudo) by default.
+
+Note: as of March 20, 2021, HIDAPI (`comms/hidapi`), [ ... ]
```

Comment:
Oh, sorry, `[ ... ]` wasn't a literal suggestion, it just meant "no change from this point forwards".